src/ArrayCache.php

```diff
@@ -46,4 +58,83 @@ public function clear(): bool
         $this->cache = [];
         return true;
     }
+
+    public function getMultiple($keys, $default = null)
+    {
+        $results = [];
+        foreach ($keys as $key) {
+            $value = $this->get($key, $default);
+            $results[$key] = $value;
+        }
+        return $results;
+    }
+
+    public function setMultiple($values, $ttl = null): bool
+    {
+        foreach ($values as $key => $value) {
+            $this->set($key, $value, $ttl);
+        }
+        return true;
+    }
+
+    public function deleteMultiple($keys): bool
+    {
+        foreach ($keys as $key) {
+            $this->delete($key);
+        }
+        return true;
+    }
+
+    public function has($key): bool
+    {
+        return isset($this->cache[$key]) && !$this->isExpired($key);
+    }
+
+    /**
+     * Checks whether item is expired or not
+     * @param $key
+     * @return bool
+     */
+    private function isExpired($key): bool
```

Comment:
```suggestion
    private function isExpired(string $key): bool
```

Should we type-hint it?